bin/prover/src/prove/mod.rs

```diff
@@ -1,8 +1,17 @@
-use axum::{http::StatusCode, response::IntoResponse, routing::post, Router};
+use axum::{http::StatusCode, response::IntoResponse, routing::post,routing::get, Router};
 use podman::process::ProcessError;
 use thiserror::Error;
 
 mod state_diff_commitment;
+mod models;
+
+
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+use lazy_static::lazy_static;
+lazy_static! {
```

Comment:
use app state created in main instead of global
[here](https://github.com/visoftsolutions/ksox/blob/master/server/worker/src/api/auth/mod.rs#L29) is the example